src/App.js

```diff
@@ -6,33 +6,62 @@ import { fetchBusinesses } from './services/yelp';
 function App() {
   const [businesses, setBusinesses] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [error, setError] = useState('');
 
   // TODO -- add state for zip / search and add event listeners to the inputs
+  const [zipCode, setZipCode] = useState('97202');
+  const [searchQuery, setSearchQuery] = useState('');
 
   useEffect(() => {
     const fetchData = async () => {
       const data = await fetchBusinesses();
-      setBusinesses(data);
+      setBusinesses(data.data.businesses);
```

Comment:
It looks like your data is nested is another data object? Renders well, just making sure.